### PR TITLE
New TWiS, events moved online, app listing disclaimer

### PR DESCRIPTION
**Pull request by Vinnl**

### CODE REVIEW

File: pages/use-solid/apps.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ permalink: /use-solid/apps
 
 Please be aware that this page is still a work in progress. You wrote a Solid-compatible app that's not listed? Feel free to [add it to this list](https://github.com/solid/solidproject.org/edit/staging/pages/use-solid/apps.md).
 
+Disclaimer: The list below does not represent an endorsement by the Solid Project. The apps on the list do not undergo any testing at this time, and there is no guarantee of security or quality implied by their inclusion on this list. The goal of the list is to showcase the vibrancy of the Solid community and encourage early app builders to collaborate and learn from each other.
+
+If you believe one of the apps below is insecure or non-functional and should be removed from the list, please raise an issue [here] (https://github.com/solid/solidproject.org/issues) and tag the app's creator or maintainer. If the creator or maintainer of the app does not respond within 24 hours to indicate that they will fix the problem, a Solid Administrator will remove the app from the listing.
```
Comment (csarven) [inline pages/use-solid/apps.md]: I think 24 hours is a short notice. Perhaps a week? If no response, the Solid Administrator should review the removal request and the app. It prevents arbitrary removal requests and removing without an actual review (if creator/maintainer doesn't show up on time).

Comment (Vinnl) [reply pages/use-solid/apps.md]: Hmm, is this the only thing blocking approval for now? If so, I can revert this change, and we can discuss further how this should be improved in #161?

Comment (csarven) [reply pages/use-solid/apps.md]: It is a non-blocker for me. Okay to approve the PR. Improve in 161 sounds good.

**Comment by Vinnl** [comment]
Great, thanks! I'll follow up in that issue.

File: _posts/for-developers/apps/vocabularies/2019-01-01-05_common.md [hunk 2]
```diff
@@ -161,10 +161,10 @@ This vocabulary helps describing copyright licenses.
 
 ```turtle
 # The Obelisk ontology is available under the CC 4.0 license...
-<http://w3id.org/obelisk/ns/> cc:license <http://creativecommons.org/licenses/by/4.0/>.
+<http://w3id.org/obelisk/ns/> cc:license <https://creativecommons.org/licenses/by/4.0/>.
```
Comment (kjetilk) [inline _posts/for-developers/apps/vocabularies/2019-01-01-05_common.md]: We shouldn't change that, I think. There is a Design Issue on it: https://www.w3.org/DesignIssues/Security-NotTheS.html
and it seems the CC server does the right thing when upgrading.

File: _posts/for-developers/apps/vocabularies/2019-01-01-12_extended-creation.md [hunk 3]
```diff
@@ -263,7 +263,7 @@ obelisk: rdf:type owl:Ontology ;
     vann:preferredNamespacePrefix "obelisk" ;
     vann:preferredNamespaceUri <http://w3id.org/obelisk/> ;
     # Ownership information
-    dcterms:license <http://creativecommons.org/licenses/by/4.0/> ;
+    dcterms:license <https://creativecommons.org/licenses/by/4.0/> ;
```
Comment (kjetilk) [inline _posts/for-developers/apps/vocabularies/2019-01-01-12_extended-creation.md]: here too

File: _posts/for-developers/apps/vocabularies/2019-01-01-16_publish-rdf.md [hunk 4]
```diff
@@ -38,7 +38,7 @@ We already described some meta that should be attached to a vocabulary when cove
...
     - [terms:creator](http://purl.org/dc/terms/creator) and [terms:contributor](http://purl.org/dc/terms/contributor) the creator(s) and contributor(s) of the vocabulary. All the better if they have webid that can be linked to!
-    - [terms:rights](http://purl.org/dc/terms/rights) or [cc:license](http://creativecommons.org/ns#licence) to describe the property rights or licence under which the vocabulary is available. Some widely used licenses are themselves identified by IRI, so you can directly reference them: [creative commons](http://creativecommons.org/licenses/by/4.0/), [Apache 2.0](https://opensource.org/licenses/Apache-2.0), [MIT](https://opensource.org/licenses/mit-license)...
+    - [terms:rights](http://purl.org/dc/terms/rights) or [cc:license](http://creativecommons.org/ns#licence) to describe the property rights or licence under which the vocabulary is available. Some widely used licenses are themselves identified by IRI, so you can directly reference them: [creative commons](https://creativecommons.org/licenses/by/4.0/), [Apache 2.0](https://opensource.org/licenses/Apache-2.0), [MIT](https://opensource.org/licenses/mit-license)...
```
Comment (kjetilk) [inline _posts/for-developers/apps/vocabularies/2019-01-01-16_publish-rdf.md]: And in this file

File: assets/misc/tutorials/obelisk.ttl [hunk 5]
```diff
@@ -17,7 +17,7 @@ obelisk: rdf:type owl:Ontology ;
     vann:preferredNamespacePrefix "obelisk" ;
     vann:preferredNamespaceURI <http://w3id.org/obelisk/> ;
     # Ownership
-    dcterms:license <http://creativecommons.org/licenses/by/4.0/> ;
+    dcterms:license <https://creativecommons.org/licenses/by/4.0/> ;
```
Comment (kjetilk) [inline assets/misc/tutorials/obelisk.ttl]: and here

**Comment by Vinnl** [comment]
@kjetilk Note that "change" here is not changing the data model of an actual app, it's changing the example Turtle files. We didn't know which one is considered the canonical one when the tutorial was written, and the HTTP version started throwing errors, so we assumed the HTTPS one was the canonical one and thus is the one that should be listed in the documentation.

So bottom line: we should tell people to use whatever URL is the canonical one, but unfortunately we have no idea how to tell which one that is, so we made this assumption. Happy to hear suggestions for a better way to determine the right one!

**Comment by kjetilk** [comment]
Yup! My hunch is "don't change", especially as long as there is a redirect. @RubenVerborgh , what do you think?

**Comment by RubenVerborgh** [comment]
Wouldn't change.

**Comment by Vinnl** [comment]
@kjetilk There was no redirect, there was an error. Why was the previous version better? Should we be encouraging people to use the potentially non-canonical URL?

**Comment by kjetilk** [comment]
> @kjetilk There was no redirect, there was an error. 

Oh, yeah. Weird, I get `521` when I `curl` it... I don't know why that happens, I haven't seen that before. I don't have that much of a good practical advice, I must admit, I have to leave that to the front-enders. I don't know how feasible it it is to get the world to implement a reasonable upgrade path.

> Why was the previous version better? Should we be encouraging people to use the potentially non-canonical URL?

If you look at the RDFa source of that page, it actually uses the non-TLS URI to refer to itself:
```html
<head about="http://creativecommons.org/licenses/by/4.0/">
``` 
so, the canonical URL is that, I think.

> (And again, these aren't "real" Turtle files; they're examples.)

Yeah, but people will cut'n'paste, and so things could get messy.

**Comment by Vinnl** [comment]
> If you look at the RDFa source of that page, it actually uses the non-TLS URI to refer to itself:

Ah, alright - that seals the deal for me. Then we'll just refer to a broken link, and have our broken link detector ignore that one - the onus for fixing it would be on the domain owner, so not much we can do here I think. I'll submit a PR.

**Comment by csarven** [comment]
>so, the canonical URL is that, I think.

Just wanted to note that the RDFa bits in CC's HTML is pretty old.. RDF 1.0 old... and well before the mass http->https transition.

Note also that while http://creativecommons.org/ns redirects to a https URL, the contents still use the http URI.

Note also things like:
```html
<div xmlns:cc="https://creativecommons.org/ns#" about="https://creativecommons.org">
```

or

```turtle
<http://creativecommons.org/licenses/by/4.0/>
  <http://purl.org/dc/elements/1.1/creator> <https://creativecommons.org/> .
```

The template is not consistent and was edited by different people over time. I wouldn't rely on that as canonical or exemplary because it doesn't seem to be maintained - at least the RDF layer.

See also: http://rdf.greggkellogg.net/distiller?command=serialize&url=https:%2F%2Fcreativecommons.org%2Flicenses%2Fby%2F4.0%2F&raw

You'll also note that some of the triples about the license language translations use https whereas the "canonical" uses http.

Personally I'd stick to `https://creativecommons.org/licenses/by/4.0/` and `http://creativecommons.org/ns`. I'd be more strict on the RDF ns than the document that's generally intended for humans (even though it has some RDFa in it). If you want consistency for the sake of it... just use http but really.. the examples are just informal/nonnormative text. It is "okay" to transition to https for those kind of resources IMO.

**Comment by kjetilk** [comment]
Allright, I shall admit that I don't really have a clue, it was just a flag to raise for me.

**Pull request merged**